blog/docs/articles/knative-eventing-vision.md

```diff
@@ -0,0 +1,88 @@
+
+### Transport encryption
+
+Currently, event delivery within the cluster is unencrypted. This limits the types of events which can be transmitted to those of low compliance value (or a relaxed compliance posture) or, alternatively, forces administrators to use a service mesh or encrypted CNI to encrypt the traffic, which poses many challenges to Knative Eventing adopters.
+
+Knative Brokers and Channels will provide HTTPS endpoints to receive events. Given that these endpoints typically do not have public DNS names (e.g. svc.cluster.local or the like), these need to be signed by a non-public CA (cluster or organization specific CA).
+
+Event producers will be able to determine the CA root used to sign the Broker or Channel certificate from the Knative resource’s metadata.
+
+For more information, please refer to the [feature track document](https://docs.google.com/document/d/1H-x_oji8LqkCyd7tlsSyclmUe7FAmEJPgRxOU_0pkn8/edit?resourcekey%253D0-lzDIPJsZOP3G17QE_g1lHw#heading%253Dh.8eg1on3fge8w).
+
+### Sender identity
+
+Currently, event delivery within the cluster is unauthenticated, and an event consumer cannot determine the identity of any sender.
+
+By sending the identity of the event sender along with the events, event receivers can start implementing access control features.
+
+For more information, please refer to the [feature track document](https://docs.google.com/document/d/1e7UgNTkL0Br5Da09Rg2ieVmhKJo4VXuBj-mHT9NCujY/edit).
+
+### Authorization and admission policies
+
+Event-driven architecture is meant to lower organizations' silos and barriers, make systems more resilient and ultimately increase business agility, however, this is not possible without event admission policies and safeguards for security and data quality.
+
+Event admission policies will address the need to have authorization policies regarding who is allowed to send events to a given event hub, like a Knative Broker or Channel, as well as what they accept as valid content for events sent to them (schemas, etc).
+
+Authorization and admission policies will be the feature that will bridge the secure event processing topic to the event discovery topic, below.
+
+## Improved Event discovery
+
+Event discovery helps developers to understand which events they can consume and how they look like, without having to dig through various levels of service documentation.
+
+Knative defines an EventType API which allows for discovery of the types of events that are available in the Knative Eventing system. This helps developers understand what kinds of events can be listened to and processed, which can be particularly helpful in systems with a large number of events being produced.
+
+However currently the EventType is unfortunately underused and is limited to Source-Duck compliant sources, in combination with the Knative Broker API. 
```

Comment:
Any links to "Source-Duck compliant sources" ? I guess only 5-6 people will get what this means in the entire world :D

Reply:
But it is a fact, since that auto-create only works under some rules.

Any suggestion to say this differently?

Reply:
Maybe something like 

"However currently the EventType is unfortunately underused and is limited to sources which are compliant with / implement the Knative [Source Ducktype](https://github.com/knative/pkg/blob/aef227e72ead466e1ff7a5dddca2c171b0d5f128/apis/duck/v1/source_types.go#L40), in combination with the Knative Broker API"